iphone/Maps/LocalizedStrings/zh-Hant.lproj/Localizable.stringsdict

```diff
@@ -46,8 +41,8 @@
 			<key>NSStringFormatValueTypeKey</key>
 			<string>d</string>
 			<key>other</key>
-			<string>%d 個軌跡</string>
+			<string>%d 軌跡</string>
```

Comment:
Plurals are screwed up in the current translation system.

https://github.com/organicmaps/organicmaps/pull/10321#issuecomment-2676771426